client/client.go

```diff
@@ -58,31 +60,27 @@ func (kc *KalaClient) decode(body io.Reader, target interface{}) error {
 }
 
 func (kc *KalaClient) url(parts ...string) string {
-	for i, part := range parts {
-		if part == "" {
-			continue
-		}
-		if init, last := part[:len(part)-1], part[len(part)-1]; last == '/' {
-			parts[i] = init
-		}
-	}
 	return strings.Join(append([]string{kc.apiEndpoint}, parts...), "/") + "/"
```

Comment:
that logic is no longer needed. defining and using https://github.com/ajvb/kala/pull/126/files#diff-1cd41a2611da49ce63630d5a51bd82c5R22 strips the trailing / from the only url part that had one. using `path.Join` makes `client.GetJob("/abc")` equivalent to `client.GetJob("abc")`. up to you if you want to support that, but I wouldn't